subjects/fillit.en.md

```diff
@@ -0,0 +1,109 @@
+## fiil it
+
+### Objectives
+
+Develop a program that receives only one parameter, a txt file which will contain a list of [**Tetrominos**](https://en.wikipedia.org/wiki/Tetromino) to assemble them in order to create the smallest square possible.
+
+### Instructions
+
+All files referring to the project, must be in the same folder.
+
+The program must:
+
+- Be written in **GO**
+- Be compiled to be runned
+- Assemble all of the **Tetrominos** in order to create the smallest square possible
```

Comment:
In the wikipedia article, the plural of tetromino seems to be more often tetrominoes.
I also suggest not to treat the word tetromino as a special word, by that I mean not putting it in bold or capital letters.